src/main/java/com/prgrms/boardjpa/User/domain/User.java

```diff
@@ -0,0 +1,57 @@
+package com.prgrms.boardjpa.User.domain;
+
+import com.prgrms.boardjpa.global.ErrorCode;
+import com.prgrms.boardjpa.global.domain.BaseEntity;
+import com.prgrms.boardjpa.global.exception.InvalidDomainConditionException;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.springframework.util.StringUtils;
+
+import javax.persistence.*;
+import java.util.regex.Pattern;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Table(name = "users")
+@Entity
+public class User extends BaseEntity {
+
+    private static final Pattern NAME_REGEX_PATTERN = Pattern.compile("^[가-힣a-zA-Z0-9]+$");
+    private static final int MINIMUM_AGE = 1;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
```suggestion
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
```

h2를 사용하고 있으니 GenerationType.SEQUENCE로 명확하게 설정해보시는 건 어떨까요~?